homeassistant/components/osoenergy/strings.json

```diff
@@ -79,5 +79,143 @@
         "name": "Profile local"
       }
     }
+  },
+  "services": {
+    "get_profile": {
+      "name": "Get Heater Profile",
```

Comment:
Recommend using sentence case (only capitalize the first word) for the names.  This would be in line with items above, and from what I can tell it seems to be the preferred style.